lib/api_catalogue.rb

```diff
@@ -10,6 +12,28 @@ def self.from_csv(catalogue_csv:, organisation_csv:)
     new(apis: apis, organisations: organisations)
   end
 
+  def self.from_url(url)
+    V1AlphaFetcher.fetch(url)
+  rescue RuntimeError => e
+    puts e.message
+    ApiCatalogue.new(apis: [], organisations: [])
+  end
+
+  def self.merge(*api_catalogues)
```

Comment:
did we consider making this an instance method, rather than a static one? Not sure if it would be nicer, but that's what I probably would've reached for first